mne/io/brainvision/brainvision.py

```diff
@@ -477,20 +477,32 @@ def _get_vhdr_info(vhdr_fname, eog, misc, scale, montage):
     misc = list(misc_chs.keys()) if misc == 'auto' else misc
 
     # create montage
-    if montage is True:
-        from ...transforms import _sphere_to_cartesian
+    if cfg.has_section('Coordinates') and montage is None:
+        from ...transforms import _sph_to_cart
         from ...channels.montage import Montage
         montage_pos = list()
         montage_names = list()
+        to_misc = list()
         for ch in cfg.items('Coordinates'):
-            montage_names.append(ch_dict[ch[0]])
+            ch_name = ch_dict[ch[0]]
+            montage_names.append(ch_name)
             radius, theta, phi = map(float, ch[1].split(','))
             # 1: radius, 2: theta, 3: phi
-            pos = _sphere_to_cartesian(r=radius, theta=theta, phi=phi)
+            pol = np.deg2rad(theta)
+            az = np.deg2rad(phi)
+            pos = _sph_to_cart(np.atleast_2d([radius * 85., az,
+                                              pol])).squeeze()
```

Comment:
why the 0.85 and the squeeze?

Reply:
Sorry for the silence the last days.

Not sure I understand: `_sph_to_cart` asserts `cart.ndim == 2 and cart.shape[1] == 3` and the output is always 2d. How do you propose to get around this?

Reply:
`_sph_to_cart([[radius * 85., az, pol]])[0]`

Reply:
I don't think list of lists will work since `_sph_to_cart` uses `ndim` and `shape` methods of the input array. How about `_sph_to_cart(np.array([[radius * 85., az, pol]]))[0]`?

Reply:
LGTM

Reply:
(or add an `asarray` call to `_sph_to_cart`)

Reply:
the mystery of the `85` factor should be resolved, see the inquiry at the PR that introduced it: https://github.com/mne-tools/mne-python/pull/4109#issuecomment-513444467